libraries/apollo-gradle-plugin/testProjects/kotlin-plugin-version-min/build.gradle.kts

```diff
@@ -17,9 +17,9 @@ configure<ApolloExtension> {
 }
 
 tasks.withType<KotlinCompile>().configureEach {
-  kotlinOptions {
+  compilerOptions {
     // Required for Kotlin < 1.6.10
     // See https://kotlinlang.org/docs/whatsnew1620.html#compatibility-changes-in-the-xjvm-default-modes
-    freeCompilerArgs += "-Xjvm-default=all"
+    freeCompilerArgs.add("-Xjvm-default=all")
```

Comment:
I forgot to remove this in the AGP9 update but this is actually not used, I'll remove it.